site/app/templates/forum/GeneratePostList.twig

```diff
@@ -1,6 +1,7 @@
 {% set show_unresolve = false %}
 <data id="current-thread" value="{{ post_data[0].thread_id }}"></data>
-
+<a class="skip-btn" href="#thread_box_link_{{thread_id != "" ? thread_id : first_post_id}}">Back to thread list</a>
+<a class="skip-btn key_to_click" href="#" onclick="$('#reply_box_{{post_box_id}}').focus(); return false;">Skip to reply</a>
```

Comment:
You should not use `href="#"` in this context, having an empty element indicates to the browser to jump to the top of the page.

If an anchor tag is not being used to point to a valid URL or element it should not use the href tag, however this will mean you will have to set tabindex manually and potentially set some of the styling

I recommend passing in the id of the "post to thread" box (the one at the very bottom) through twig and setting the href to point at that element like the rest of the navigation skip links.